cmd/status-backend/server/server.go

```
@@ -53,12 +53,28 @@ func (s *Server) signalHandler(data []byte) {
 	s.lock.Lock()
 	defer s.lock.Unlock()
 
+	var disconnected []*websocket.Conn
+
 	for connection := range s.connections {
-		err := connection.WriteMessage(websocket.TextMessage, data)
+		err := connection.SetWriteDeadline(time.Now().Add(5 * time.Second))
+		if err != nil {
+			log.Error("failed to set write deadline", "error", err)
+			disconnected = append(disconnected, connection)
+			_ = connection.Close()
```

Why ignore error on `Close`?

---

Or is there a way to restore the connection to its normal state? If you mean log the err, I think it's that useful 🤔

---

Yeah, just log the error 👍